src/css/molecules/card.scss

```diff
@@ -381,6 +381,9 @@
   }
 }
 
+/* -------------------------------------------------------------------------- */
+// NEW CUSTOM CSS FOR CARDS
+
```

Comment:
Left overs?

Reply:
Yep - i'll be dropping it when am sure we don't need any bits of it. There'll be a dedicated cleanup ticket